Sources/TabBar/View/TabBar.swift

```diff
@@ -48,13 +55,15 @@ public struct TabBar<TabItem: Tabbable, Content: View>: View {
         self.content = content()
         
         self._items = .init(initialValue: .init())
+        self.visibility = visibility
     }
     
-    public init(selection: Binding<TabItem>, @ViewBuilder content: () -> Content) {
+    public init(selection: Binding<TabItem>, visibility: Binding<TabBarVisibility>, @ViewBuilder content: () -> Content) {
```

Comment:
This line actually too long, add line-wrapping as I did in the private initializer